packages/flutter_tools/test/integration.shard/test_data/hot_reload_project.dart

```diff
@@ -81,10 +73,20 @@ class HotReloadProject extends Project {
     }
   }
 
-  void printHotReloadWorked() {
+  Future<void> printHotReloadWorked() async {
     // The call to this function is uncommented by a test to verify that hot
     // reloading worked.
     print('(((((RELOAD WORKED)))))');
+
+    // We need to insist here for `const` Apps, so print statements don't
+    // get lost between the browser and the test driver.
+    // See: https://github.com/flutter/flutter/issues/86202
```

Comment:
IDK what's going on here, but if it fixes the test, I can live with that :P

Reply:
Check the linked issue. This loop used to run `runApp` every second, but we discovered that we just need to run the "hot restart worked" bits :)